### DISCUSSION TITLE
请问 VLESS+GRPC+REALITY 搭配 nginx 怎么配置？ 这个配置能否开启CDN 代理，谢谢。

### DISCUSSION THREAD

**Discussion by wpyok500** [lxhao61/integrated-examples#182] [Q&A] [answered]
请问 VLESS+GRPC+REALITY 搭配 nginx 怎么配置？ 这个配置能否开启CDN 代理，谢谢。

烦请给个配置模板，谢谢。

**Comment by lxhao61** [comment, marked as answer]
参考 [Xray(VLESS+Vision+REALITY)](https://github.com/lxhao61/integrated-examples/tree/main/Xray(VLESS%2BVision%2BREALITY))。不能 CDN 代理。
